rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
```

Comment:
- [ ] Getters and Setters (attr_accessor) for @book and @person instance variables should be present, not Getters only as you did, Please fix that.